### DISCUSSION TITLE
Variantof items don't inherit correct path if you mod their prototype

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#14308] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

E.g. i want to replace some sprites in nav terminal, i override it and it works but, navterminalwrecked inherits wrong path and breaks
Instead of `%ModDir%/Sonar/pingCircle.png` it gets `Content/ContentPackages/Sonar/pingCircle.png`

![Screenshot_2](https://github.com/user-attachments/assets/d8245c54-05cf-41fc-acad-67b55101ce2e)

It still can be done by overriding navterminalwrecked fully, without variantof 
![Screenshot_3](https://github.com/user-attachments/assets/a92dfc87-b786-4410-abc0-6fd76fb885d5)

### Reproduction steps

Take this mod https://steamcommunity.com/sharedfiles/filedetails/?id=3292155481
Go to sub editor
Try placing wrecked nav terminals

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/issues/13302. This should be fixed in the latest unstable, sorry for the inconvenience!
